src/store/utils.js

```diff
@@ -34,7 +34,7 @@ export const retryIfSourceChainHeadMoved = async call => {
       val.type === 'error' &&
       val.payload &&
       val.payload.message &&
-      val.payload.message.includes('head moved')
+      val.payload.message.includes('source chain head has moved')
```

Comment:
This is interesting and important feedback back to holochain team.  We probably can't rely on that text staying the same.  I wonder when we start introducing error numbers.